web/src/app/modules/shared/components/smart/editor/editor.component.ts

```diff
@@ -137,6 +137,8 @@ export class EditorComponent
   }
 
   ngOnDestroy() {
-    this.subscriptionTheme.unsubscribe();
+    if (this.subscriptionTheme) {
+      this.subscriptionTheme.unsubscribe();
```

Comment:
Not a biggie, but `this.subscriptionTheme.unsubscribe();` would be more elegant and easier to read.

Reply:
The test in karma is not guaranteed to have subscribed to the service on tear down.

Reply:
Sorry, that was supposed to be `this.subscriptionTheme?.unsubscribe();` 😊